src/DotNet.Consolidate.Tests/TestData/TestSolution/Directory.build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
There is no need to update the test solution. We do not build it anyway.

Reply:
Reverted this change